diff --git a/node_modules/uglify-js/bin/uglifyjs b/node_modules/uglify-js/bin/uglifyjs
index 63b2f26..d12131f 100755
--- a/node_modules/uglify-js/bin/uglifyjs
+++ b/node_modules/uglify-js/bin/uglifyjs
@@ -513,7 +513,7 @@ function done() {
                 var base64_string = new Buffer(SOURCE_MAP.toString()).toString('base64');
                 output += "\n//# sourceMappingURL=data:application/json;charset=utf-8;base64," + base64_string;
             } else {
-                fs.writeFileSync(ARGS.source_map, SOURCE_MAP, "utf8");
+                fs.writeFileSync(ARGS.source_map, SOURCE_MAP.toString(), "utf8");
                 var source_map_url = ARGS.source_map_url || (
                     P_RELATIVE
                         ? path.relative(path.dirname(OUTPUT_FILE), ARGS.source_map)
